Makefile

```diff
@@ -190,6 +190,25 @@ BUILD_SYSTEMS_TESTS += mvn
 endif
 endif
 
+ifeq ($(BUILD_DOTNET_ANALYZERS),yes)
```

Comment:
Is this `ifeq` needed? We provide compile-time options in `./configure` to disable the clang or Java frontends because sometimes it makes sense for the user, but these tests are independent of any frontends since they rely on `infer-analyzejson`. If you remove the `ifeq` or run `make BUILD_DOTNET_ANALYZERS=yes` you should be able to run these tests as part of `make test` or individually with, eg, `make BUILD_DOTNET_ANALYZERS=yes direct_dotnet_box_test`.

Reply:
Got it. Thanks.